src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -48,6 +48,7 @@
 #include <daos_prop.h>
 #include <daos/common.h>
 #include <daos/dfs_lib_int.h>
+#include <../common/dav/util.h>
```

Comment:
You shouldn't be using relative includes but also, why is the dav header file needed at all?

If this genuinely is needed then add `src/common` to the `-I` path in the scons file and include this as `<dav/util>`.

Reply:
The issue is fixed. Now atomic_compare_exchange_weak() is used to replace util_bool_compare_and_swap64(). Thank you!